repo.go

```diff
@@ -1092,3 +1092,17 @@ func (r *Repo) timestampFileMeta(roleFilename string) (data.TimestampFileMeta, e
 	}
 	return util.GenerateTimestampFileMeta(bytes.NewReader(b), r.hashAlgorithms...)
 }
+
+func (r *Repo) Payload(roleFilename string) ([]byte, error) {
+	role := strings.TrimSuffix(roleFilename, ".json")
+	if !roles.IsTopLevelRole(role) {
+		return nil, ErrInvalidRole{role}
+	}
+
+	s, err := r.SignedMeta(roleFilename)
```

Comment:
It's a small nit, but I noticed the `tuf payload` command implicitly expects a role filename instead of what's in its description - i.e., `root.json` instead of `root`. Adding a test for that would be helpful too 👍

Reply:
Added a test that we get `ErrMissingMetadata` and changed help text to expect `<metadata>` instead of `<role>`